libs/lz-string.js

```diff
@@ -496,6 +496,8 @@ var LZString = {
 
 if (typeof define === 'function' && define.amd) {
   define(function () { return LZString; });
+} else if (typeof chrome !== "undefined" && typeof chrome.extension !== "undefined") {
```

Comment:
if we are running on chrome and on a chrome extension context, the scope is the "UnsafeWindow" where the extension is running